### PR TITLE
djvulibre: Add version 3.5.27-4.11

### PR DESCRIPTION
**Pull request by niheaven**

### CODE REVIEW

File: bucket/djvulibre.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+    "version": "3.5.27-4.11",
+    "description": "DjVu library.",
```
Comment (Ash258) [inline bucket/djvulibre.json]: This is not much optimal description.

```suggestion
    "description": "Web-centric format and software platform for distributing documents and images.",
```

File: bucket/djvulibre.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    "shortcuts": [
+        [
+            "djview.exe",
+            "DjView"
+        ]
+    ],
+    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\uninst.exe\" -Recurse -Force",
+    "checkver": {
+        "url": "https://sourceforge.net/projects/djvu/rss?path=/DjVuLibre_Windows",
+        "regex": "/(?<lib>[0-9.]+)(?:\\+(?<viewer>[0-9.]+))?/DjVuLibre",
```
Comment (Ash258) [inline bucket/djvulibre.json]:
```suggestion
        "regex": "/(?<lib>[\\d.]+)(?:\\+(?<viewer>[\\d.]+))?/DjVuLibre",
```

**Comment by niheaven** [comment]
> DjVuLibre includes a standalone viewer, a browser plug-in and command line tools.

The browser plugin isn't here, and the viewer is `djview.exe` which could used as a CLI tool. And since DjVuLibre is used mainly as CLI tools, putting it in main should be better.

**Comment by Ash258** [comment]
I guess this is pretty much same as git, pwsh and similar tools.

**Comment by niheaven** [comment]
GUI tool is just a supplement.

**Pull request merged**